doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -3431,7 +3431,7 @@ \subsubsection{Inputs}\label{inputs-9-014}
 
 \paragraph{Field: Condenser Outlet Node Name}\label{field-condenser-outlet-node-name-8}
 
-This required alpha field contains the identifying name for the chiller condenser side outlet node.
+This alpha field contains the identifying name for the chiller condenser side outlet node. It is needed for WaterCooled chillers but not for AirCooled.
```

Comment:
Since you'll be editing the file anyway, would be better to say "required" instead of "needed" here and below.